src/renderer/api/endpoints/role-binding.api.ts

```diff
@@ -24,15 +24,17 @@ import { KubeObject } from "../kube-object";
 import { KubeApi } from "../kube-api";
 import type { KubeJsonApiData } from "../kube-json-api";
 
-export interface IRoleBindingSubject {
-  kind: string;
+export type RoleBindingSubjectKind = "Group" | "ServiceAccount" | "User";
+
+export interface RoleBindingSubject {
+  kind: RoleBindingSubjectKind;
   name: string;
-  namespace?: string;
+  namespace: string;
```

Comment:
does this mean that `Group` or `User` must always be namespaced?

Reply:
For `RoleBinding` subjects yes, they need to be the same as the namespace that the `RoleBinding` is in.

Reply:
> For `RoleBinding` subjects yes, they need to be the same as the namespace that the `RoleBinding` is in.

That's not exactly true, the namespace just indicates in what namespace the service account does exist. It does not need to be the same namespace where RoleBinding is created.

`user` and `group` does not need namespace, like this example [example](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-example):
```
apiVersion: rbac.authorization.k8s.io/v1
# This role binding allows "jane" to read pods in the "default" namespace.
# You need to already have a Role named "pod-reader" in that namespace.
kind: RoleBinding
metadata:
  name: read-pods
  namespace: default
subjects:
# You can specify more than one "subject"
- kind: User
  name: jane # "name" is case sensitive
  apiGroup: rbac.authorization.k8s.io
roleRef:
  # "roleRef" specifies the binding to a Role / ClusterRole
  kind: Role #this must be Role or ClusterRole
  name: pod-reader # this must match the name of the Role or ClusterRole you wish to bind to
  apiGroup: rbac.authorization.k8s.io

Reply:
Ah okay, will revert that. From my testing it seemed like it was always there.